nix/README.md

```diff
@@ -0,0 +1,149 @@
+# Prism Launcher Nix packaging
+
+## Installing a stable release (nixpkgs)
+
+Prism Launcher is packaged in [nixpkgs](https://github.com/NixOS/nixpkgs/) since 22.11.
+
+See [Package variants](#package-variants) for a list of available packages.
+
+## Installing a development release (flake)
+
+### Using the overlay
+
+After adding `github:PrismLauncher/PrismLauncher` to your flake inputs, you can add the `default` overlay to your nixpkgs instance.
+
+Example:
+
+```nix
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    prismlauncher = {
+      url = "github:PrismLauncher/PrismLauncher";
+      # Optional: Override the nixpkgs input of prismlauncher to use the same revision as the rest of your flake
```

Comment:
this should probably mention that overriding nixpkgs can break reproducibility with our flake